lib/fennec/application.ex

```diff
@@ -3,12 +3,26 @@ defmodule Fennec.Application do
 
   use Application
 
+  import Supervisor.Spec
+
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
 
-    children = []
 
     opts = [strategy: :one_for_one, name: Fennec.Supervisor]
-    Supervisor.start_link(children, opts)
+    {:ok, pid} = Supervisor.start_link(servers(), opts)
+  end
+
+  defp servers do
+    :fennec
+    |> Application.get_env(:servers, [])
```

Comment:
Piping an atom doesn't feel like data transformation. But it's my personal feeling. Don't treat as a MUST